datafusion-examples/Cargo.toml

```diff
@@ -75,6 +75,6 @@ serde_json = { workspace = true }
 tempfile = { workspace = true }
 tokio = { workspace = true, features = ["rt-multi-thread", "parking_lot"] }
 # 0.10 and 0.11 are incompatible. Need to upgrade tonic to 0.11 when upgrading to arrow 51
```

Comment:
```suggestion
```

No longer need this comment

Reply:
Removed in 456e2fe1892c9f5e91bde791450b07be12b96d9b